### PR TITLE
quick search footer and tweaks

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

image

image

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2688** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2688" target="_blank">Graphite</a> 
    * **PR #2689** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2689" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2689?utm_source=stack-comment).

File: src/components/QuickProjectSearch.tsx [hunk 1]
```diff
@@ -142,73 +147,109 @@ export default function QuickProjectSearch() {
...
+              </div>
+            )}
+          </div>
+
+          <div className="mt-5 flex gap-6 border-t border-r-0 border-b-0 border-l-0 border-solid border-t-smoke-200 bg-smoke-75 py-4 px-5 text-xs dark:border-t-slate-300 dark:bg-slate-600">
+            <span>
+              <KeyboardButton>
+                <EnterOutlined />
+              </KeyboardButton>{' '}
+              to select
```
Comment (johnnyd-eth) [inline src/components/QuickProjectSearch.tsx:230]: `<Trans>` tags here

File: src/components/QuickProjectSearch.tsx [hunk 2]
```diff
@@ -142,73 +147,109 @@ export default function QuickProjectSearch() {
...
+            </span>
+            <span>
+              <KeyboardButton>
```
Comment (johnnyd-eth) [inline src/components/QuickProjectSearch.tsx:233]: <Trans> tags here

**Review by johnnyd-eth — APPROVED** [review]
LGTM, just missing some `<Trans>`. Will approve in advance

**Pull request merged**